scopes/harmony/aspect-loader/core-aspects.ts

```diff
@@ -22,10 +22,18 @@ function getAspectDirFromPath(id: string, pathsToResolveAspects?: string[]): str
   return dirPath;
 }
 
-export function getAspectDir(id: string): string {
+export function getAspectDir(id: string, currentBitDir?: string | null): string {
   const aspectName = getCoreAspectName(id);
   let dirPath;
 
+  if (currentBitDir) {
+    dirPath = resolve(currentBitDir, 'node_modules/@teambit', aspectName);
+    if (!existsSync(dirPath)) {
+      throw new Error(`unable to find ${aspectName} in ${dirPath}`);
+    }
+    return dirPath;
+  }
+
```

Comment:
@zkochan why not just pass the `currentBitDir` as the second arg to the `getAspectDirFromPath` below